tests/framework/MCMC/test_metropolis.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="silent">
+  <TestInfo>
+    <name>framework/MCMC.Metropolis</name>
+    <author>wangc</author>
+    <created>2020-06-26</created>
+    <classesTested>MCMC.Metropolis</classesTested>
+    <description>
+      Test the Markov Chain Monte Carlo algorithm: Metropolis Sampling
+      A 2-D multivariate normal distribution is used as the likelihood function.
+      mean: [5, 5], cov=[[1, 0.9], [0.9, 1]]
+      Both input parameters have the standard normal distribution as their prior distribution.
+      The proposal distributions for the input variables are also standard normal distribution.
+      "TargetEvaluation" is used to collect the inputs and outputs from the likelihood model.
```

Comment:
since this goes into a latex file, I would use the ``  '' construct for the quotes

Reply:
fixed